mempool/clist_mempool.go

```diff
@@ -662,6 +662,39 @@ func (mem *CListMempool) recheckTxs() {
 	mem.proxyAppConn.FlushAsync()
 }
 
+func (mem *CListMempool) GetNextTxBytes(remainBytes int64, remainGas int64, starter []byte) ([]byte, error) {
```

Comment:
add a comment saying this O(n) method for finding next tx matching priority is temporary and in the future we can optimize with balance tree and reduce complexity to O(logn) and even O(1) by keeping iterating state.